src/test.rs

```diff
@@ -1068,6 +1143,28 @@ mod tests {
         assert_eq!(&result.id, "12345");
     }
 
+    #[actix_rt::test]
+    async fn test_body_json() {
```

Comment:
Since the function is called `read_body_json()` it is probably a good idea to call this test `test_read_body_json()`.

Reply:
It's the same as `read_response_json` and `test_response_json`, we can follow-up later.